assets/js/modules/analytics/dashboard/dashboard-widget-top-acquisition-sources-table.js

```diff
@@ -46,7 +46,10 @@ import { STORE_NAME } from '../../../googlesitekit/datastore/user/constants';
 const { useSelect } = Data;
 
 function AnalyticsDashboardWidgetTopAcquisitionSources( { data } ) {
-	const dateRange = useSelect( ( select ) => select( STORE_NAME ).getDateRange() );
+	const dateRange = useSelect( ( select ) => {
+		const store = select( STORE_NAME );
+		return store ? store.getDateRange() : undefined;
```

Comment:
Yes, it is necessary because when it runs the first time `select( STORE_NAME )` returns `undefined`. Probably, because that store isn't registered yet :thinking:

Reply:
I'm trying to reproduce this but it works as expected for me without this conditional. Also, this component JS is bundled in `googlesitekit-module.js` so the `googlesitekit-datastore-user` should always be loaded by then.
![image](https://user-images.githubusercontent.com/1621608/87334081-cc0ce080-c546-11ea-93e8-127cef778f0c.png)

Can you check your build again perhaps?

Reply:
Hm... it works as epxected now.... Weird... Ok, reverted to be as regular.

Reply:
It's been fixed.